cacheops/redis.py

```diff
@@ -58,7 +100,10 @@ def __setattr__(self, name, value):
 try:
     redis_conf = settings.CACHEOPS_REDIS
     redis_replica_conf = settings.CACHEOPS_REDIS_REPLICA
-    redis_replica = redis.StrictRedis(**redis_replica_conf)
+
+    class SafeReplicaRedis(redis.StrictRedis):
```

Comment:
Yes. The thing is when error occured during the query to replica we catch it and make request to master. So we don't need to handle replica's exceptions in the same way as master's.